### PR TITLE
Make tls_conn_timeout test more reliable in CI - fixes autest on master

### PR DESCRIPTION
**Pull request by shinrich**
tls_conn_timeout test is not passing in the master autest build.  Streamlining some things with the test here.  Two things that should specifically improve stability.  Modeling the ssl-delay-server directly rather than invoking it with spreadsheet so we can delay test until we know that the delay-server's ports are ready.

Also removing the per test case timeouts.  I was trying to use them to test that the ATS timeouts weren't taking too long.  But I think with the timing vagaries in the CI, that is not going to be reliable.

### CODE REVIEW

File: tests/gold_tests/timeout/tls_conn_timeout.test.py [hunk 1]
```diff
@@ -45,60 +49,68 @@
...
 # Request the port that should timeout in the handshake
 # Should retry once
-tr = Test.AddTestRun("tr-blocking")
-tr.Setup.Copy("case2.sh")
-tr.Processes.Default.StartBefore(Test.Processes.ts, ready=When.PortOpen(ts.Variables.port))
+tr = Test.AddTestRun("tr-blocking-post")
 tr.Setup.Copy("../chunked_encoding/ssl/server.pem")
-tr.Processes.Default.Command = 'sh ./case2.sh {0} 3 0 {1} connect_blocked'.format(Test.Variables.block_connect_port, ts.Variables.port)
-tr.Processes.Default.TimeOut = 6
+tr.Processes.Default.StartBefore(Test.Processes.ts, ready=When.PortOpen(ts.Variables.port))
```
Comment (bneradt) [inline tests/gold_tests/timeout/tls_conn_timeout.test.py:61]: We shouldn't need this ready condition. I actually thought I removed all of these. The default MakeATSProcess function should set the ready that waits upon a log message from traffic server saying that it is ready to optimize traffic.

Can you please remove this ready condition from the patch?

**Comment by shinrich** [comment]
The last autest run did not pick up my latest commit.  Trying again.

**Comment by shinrich** [comment]
[approve ci autest]

**Comment by randall** [comment]
[approve ci autest]

**Comment by shinrich** [comment]
[approve ci autest]

**Comment by randall** [comment]
[approve ci freebsd]

**Comment by randall** [comment]
[approve ci freebsd]

File: tests/gold_tests/timeout/tls_conn_timeout.test.py (hunk 1 above)
Comment (shinrich) [reply tests/gold_tests/timeout/tls_conn_timeout.test.py]: Addressed.

**Pull request merged**